core/src/epicli/data/common/ansible/playbooks/roles/applications/templates/pgpool/04-deployment.yml.j2

```diff
@@ -113,6 +122,14 @@ spec:
               subPath: pgpool_postgres_password
             - name: pgpool-shared-init-dir
               mountPath: /epiphany/shared_dir
+            - name: pgpool-config-files
+              mountPath: /epiphany/bitnami/pgpool/conf/pgpool.conf
+              subPath: pgpool.conf
+            - name: pgpool-config-files
+              mountPath: /epiphany/bitnami/pgpool/conf/pool_hba.conf
+              subPath: pool_hba.conf
```

Comment:
My suggestion is to split pgpool-config-files configmap to different ones and mount config files into `/epiphany/bitnami/pgpool/conf/` at one time without `subPath` that has an affect of missing possibility to update config without pod restart.

Reply:
I think this needs to stay as it is since we also mounting here pool_passwd file in the same location but from different volume. But we can talk, i'm open for suggestions.